Rakefile

```diff
@@ -39,7 +39,11 @@ CLIENT_TOOLS = {
   '2019.8' => '19.8.6',
 }
 
-def operating_systems(collection)
+def operating_systems(collection, pkg = nil)
+  if pkg == 'puppet-bolt'
```

Comment:
Can this be an array with `.include?` so it's easy to add other packages that support 11?

Reply:
Yep. Updated.

(Technically we can remove this logic and replace it with a `when 'puppet-tools'` switch below once pdk ships for macOS 11.)